srcpkgs/bc-gh/template

```diff
@@ -1,14 +1,15 @@
 # Template file for 'bc-gh'
 pkgname=bc-gh
-version=3.1.5
+version=3.3.3
 revision=1
 wrksrc="bc-${version}"
 short_desc="Implementation of POSIX bc with GNU extensions"
 maintainer="Gavin D. Howard <yzena.tech@gmail.com>"
 license="BSD-2-Clause"
 homepage="https://github.com/gavinhoward/bc"
```

Comment:
Rather ironic, given the package name it uses, but it's moved to https://git.yzena.com/gavin/bc

Reply:
I know. But I am not sure what to do...
Should I change the homepage? The distfile? The changelog? All three? 
gh is from the authors name probably - his name is on the maintainer field 😉

Reply:
Ah, that's better. I think we can switch only changelog and homepage for now, it's better to get tarballs from GH, imo.

Reply:
Done.